data/DeepanshuRohilla15.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Deepanshu Rohilla",
+  "type": "personal",
+  "bio": "Open Source Developer, DevOps"
```

hey @DeepanshuRohilla15 please add a comma.
```suggestion
  "bio": "Open Source Developer, DevOps",
```